ghost/core/core/server/data/migrations/versions/5.27/2023-01-04-04-12-add-suppressions-table.js

```diff
@@ -0,0 +1,9 @@
+const {addTable} = require('../../utils');
```

Comment:
You need to alter the timestamp on this because it's currently ordered before the drop table

Reply:
🤣 I guess minutes wasn't precise enough